src/deserialize/parse.rs

```diff
@@ -540,6 +604,25 @@ mod test {
         );
     }
 
+    #[test]
+    fn parse_comment_in_simple_vec() {
+        let mut edn = "[11 \"2\" 3.3 ; float in simple vec\n:b true \\c]"
```

Comment:
I would have some tests that dont actually use `\n` and use strings like:
```
let mut edn = "[
  11 
  \"2\" 
  3.3 
 ;; float in simple vec
 :b 
true 
\\c
]"
```

```

Reply:
If I copy that in verbatim it passes. Is that what you intended?

Reply:
Yes

Reply:
Added a test like this